packages/subscription-widget/package.json

```diff
@@ -13,7 +13,7 @@
     "body-parser": "^1.15.2",
     "express": "^4.14.0",
     "morgan": "^1.9.1",
-    "nodemon": "^1.9.2",
+    "nodemon": "^2.0.3",
```

Comment:
Is there a reason we can't upgrade to [v2.0.15](https://github.com/remy/nodemon/releases/tag/v2.0.15)?

Reply:
Done.